doc/user/administrator/troubleshooting/pipelines/index.md

```diff
@@ -0,0 +1,962 @@
+|  3 | AssemblyAnnotation_20190725_SRR1952908 | PREPARED       | 2019-07-25 13:52:27 | 2a56795cad3c7db3   |
+|  3 | AssemblyAnnotation_20190725_SRR1952908 | SUBMITTING     | 2019-07-25 13:52:27 | 2a56795cad3c7db3   |
+|  3 | AssemblyAnnotation_20190725_SRR1952908 | ERROR          | 2019-07-25 13:52:30 | 2a56795cad3c7db3   |
++----+----------------------------------------+----------------+---------------------+--------------------+
+```
+
+This lets us see the history of the job as it was processed through IRIDA (and includes the `modified_date` giving an idea of when each stage occurred). You can see that the first two states **NEW** and **PREPARING** have a `NULL` value for `remote_analysis_id`. If the IRIDA pipeline errored in these states, there would not have been a Galaxy History created. So, you can skip trying to check the Galaxy History for more details about the job.
+
+# 3. Viewing the Galaxy History used by the IRIDA analysis pipeline
+
+## 3.1. Logging into Galaxy
+
+Once we have the Galaxy History id, we can move on to logging into Galaxy to view more details about what went wrong with the IRIDA analysis pipeline. The first step is logging into Galaxy as the same user used by IRIDA.
+
+*Note: if you do not know which Galaxy instance IRIDA is making use of, you can find this in the `/etc/irida/irida.conf` file as `galaxy.execution.url=http://GALAXY`. You can find the username as `galaxy.execution.email=galaxy-user@galaxy.org`. IRIDA uses the Galaxy API to login, which is different from the password, so you may have to check with your administrator for the password used to log into Galaxy.*
+
+![galaxy-home.png][]
+
+## 3.2. Viewing all Galaxy histories
+
+Galaxy will default to one of the histories run by IRIDA. To see all the Galaxy histories, you can go to the **Saved Histories** page.
+
+![galaxy-saved-histories.png][]
+
+This should bring us to a list of all the Galaxy histories.
+
+![galaxy-histories-list.png][]
+
+## 3.3. Viewing the correct history
+
+To view the correct History in Galaxy, we can skip directly to it using the following URL `http://GALAXY/histories/view?id=[Galaxy History id]` Where **Galaxy History id** is the id we discovered from step 1.2 (e.g., `e85a3be143d5905b`). For example, for me, going to <http://localhost:48888/histories/view?id=e85a3be143d5905b> brings up:
+
+![galaxy-view-history.png][]
+
+This is the History corresponding to the analysis pipeline in IRIDA that failed. I can now see all the failed jobs (in red). Clicking on the bug icon in one of these jobs will show me the error message (which in this case, is the same as was recorded by IRIDA).
+
+![galaxy-job-debug.png][]
+
+Clicking the **i** icon gives me more information about the Galaxy job.
+
+![galaxy-job-information.png][]
+
+Srolling to the bottom of this screen there is a lot of information about the underlying infrastructure and software:
```

Comment:
"Srolling" => "Scrolling"

Reply:
Fixed in c9fa7a06b31735e68f7a20b0706257e4966facae